README.md

```diff
@@ -2,197 +2,230 @@
 
 [![Build Status](https://travis-ci.org/vtex/toolbelt.svg?branch=master)](https://travis-ci.org/vtex/toolbelt) [![npm version](https://img.shields.io/npm/v/vtex.svg?style=flat)](https://www.npmjs.com/package/vtex) [![Greenkeeper badge](https://badges.greenkeeper.io/vtex/toolbelt.svg)](https://greenkeeper.io/)
 
-All you need to start using the VTEX platform.
+The CLI that offers all you need to start using the VTEX IO platform efficiently.
+
+#### Summary
+- [How do we empower you?](#how-do-we-empower-you)
+- [First step: Installing the VTEX Toolbelt](#installing-the-toolbelt-with-yarn)
+- [Developing an app locally](#to-develop-locally-use-vtex-link)
+- [Customize your prompt](#wanna-know-something-cool-you-can-customize-your-prompt)
+- [FAQ](#frequently-asked-questions)
+- [Contributing](#contributing)
+- [Send us feedback](#tell-us-what-you-think-about-toolbelt)
+
+# How do we empower you? 
+
+VTEX Toolbelt is a command-line program that gives you access to all of 
+the features in the VTEX platform and increases your productivity, allowing you to:
+
+- **Manage your apps quickly:** List, install and uninstall available apps in your account;
+- **Work safely on your personal environments:** List, create and remove workspaces;
+- **Easily ship changes to your final user:** Promote a workspace to `master`, making it public in your account URL;
+- **See what you are creating in real time:** Develop new apps in link mode, using automatic synchronization;
+- **Go live from the command line:** Publish apps to the VTEX App Store.
+
+# Getting started: Installing the VTEX Toolbelt
+To use the VTEX Toolbelt, you'll need to install other engines 
```

Comment:
I'm in doubt on what we should use here - `engines` or `applications`, what do you think?

Reply:
I use `engines` to be more generic. If you think that `applications` better describe what can be built with de CLI and we should not be worried about being generic, `applications`.